app/src/main/java/io/appium/uiautomator2/core/AccessibilityNodeInfoHelpers.java

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (C) 2012 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.appium.uiautomator2.core;
+
+import android.annotation.SuppressLint;
+import android.annotation.TargetApi;
+import android.graphics.Rect;
+import android.os.Build;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.test.uiautomator.UiDevice;
+import android.util.Range;
+import android.view.accessibility.AccessibilityNodeInfo;
+import android.view.accessibility.AccessibilityNodeInfo.AccessibilityAction;
+
+import io.appium.uiautomator2.utils.Logger;
+
+import static io.appium.uiautomator2.utils.Device.getUiDevice;
+import static io.appium.uiautomator2.utils.StringHelpers.charSequenceToNullableString;
+import static io.appium.uiautomator2.utils.StringHelpers.charSequenceToString;
+
+/**
+ * This class contains static helper methods to work with {@link AccessibilityNodeInfo}
+ */
+public class AccessibilityNodeInfoHelpers {
```

Comment:
Did you only rename this file or did you rename it _and_ make changes.

Reply:
I did both